### PR TITLE
Export model with ElastidcDL embedding column

### LINKED ISSUE #1595: [Serving] Export the model with ElasticDL embedding_column to the TF native SavedModel.

### PR DESCRIPTION
**Pull request by workingloong**
Fix #1595

### CODE REVIEW

File: elasticdl/python/common/model_handler.py [hunk 1]
```diff
@@ -111,6 +112,27 @@ def _replace_tf_embedding_column_with_edl(dense_features_layer):
...
+    for column in dense_features_layer._feature_columns:
+        if isinstance(column, EmbeddingColumn):
+            logger.info(
+                "Replace embedding_column {} from ElasticDL "
+                "version to TF version".format(column.name)
+            )
+            new_column = fc_lib.embedding_column(
+                column.categorical_column, dimension=column.dimension
+            )
+            new_column.set_dense_features_layer_name(dense_features_layer.name)
```
Comment (brightcoder01) [inline elasticdl/python/common/model_handler.py]: EmbeddingColumn from tensorflow don't have this method: set_dense_features_layer_name

Comment (workingloong) [reply elasticdl/python/common/model_handler.py]: Done and fix the UT.

File: elasticdl/python/tests/model_handler_test.py [hunk 2]
```diff
@@ -178,22 +200,40 @@ def test_get_model_with_embedding_layer_to_train(self):
     def test_get_model_with_embedding_column_to_train(self):
         model_inst = custom_model_with_embedding_column()
         self.assertEqual(
-            type(model_inst.layers[0]._feature_columns[1]),
+            type(model_inst.layers[2]._feature_columns[1]),
```
Comment (terrytangyuan) [inline elasticdl/python/tests/model_handler_test.py]: Assign 2 to a constant and give it a meaningful name so the code is more readable

File: elasticdl/python/tests/model_handler_test.py [hunk 3]
```diff
@@ -152,21 +164,31 @@ def setUp(self):
...
+                )
+                params.embedding_params[weight.name] = embedding_table
+            else:
+                params.non_embedding_params[weight.name] = tf.ones(
+                    weight.shape
+                )
         params.version = 100
+        return params
+
+    def _save_model(self, model):
```
Comment (brightcoder01) [inline elasticdl/python/tests/model_handler_test.py]: _fill_model_with_mock_weights_and_save_checkpoint

Comment (workingloong) [reply elasticdl/python/tests/model_handler_test.py]: _mock_model_weights_and_save_checkpoint ?

Comment (brightcoder01) [reply elasticdl/python/tests/model_handler_test.py]: `_mock_model_weights_and_save_checkpoint` is good.

**Review by brightcoder01 — APPROVED** [review]

**Pull request merged**